### ISSUE TITLE
follow your training cfg, deeplabv3+res101 can only achieve 76.777% miou

### ISSUE THREAD

**Issue by whitesockcat** [YudeWang/deeplabv3plus-pytorch#31]
- dataset:  VOC augmented segmentation dataset 
- GPUS:  4*TITAN XP
- same cfg in [`deeplabv3+voc`](https://github.com/YudeWang/deeplabv3plus-pytorch/blob/master/experiment/deeplabv3%2Bvoc/config.py)
---
I get only **76.777% miou** in val dataset, 
after run [finetune](https://github.com/YudeWang/deeplabv3plus-pytorch/tree/master/experiment/deeplabv3%2Bvocfinetuning), perform doesn't get better, only **75.815% miou**(with multi-scale & flip test)
what can I do to achieve the performance mentioned in paper?
Thx.

**Comment by YudeWang** [comment]
Hi @whitesockcat ,
Have you ever change any setting in network, such as SynchronizedBN? And the pretrained model?

**Comment by whitesockcat** [comment]
Thanks for your immediate reply. @YudeWang 
I have not used pretrained model.
My cfg is the same as [default setting](https://github.com/YudeWang/deeplabv3plus-pytorch/blob/master/experiment/deeplabv3%2Bvoc/config.py)
As for SynchronizedBN, I have not noticed. Is there any special setting for SynchronizedBN or it will be used automaticly?

**Comment by YudeWang** [comment]
@whitesockcat 
You should load ImageNet pretrained res101 and try again.
Well, I just worry about you have replaced SynchronizedBN with nn.BatchNorm.

**Comment by whitesockcat** [comment]
@YudeWang 
Thank you and I will try again.
As for SynchronizedBN, I am not quite clear. Should I change somewhere if I want to use SynchronizedBN? Besides, SynchronizedBN will perform better, right?

**Comment by YudeWang** [comment]
@whitesockcat 
SynchronizedBN has already been employed in the code and certainly it is better.

**Comment by whitesockcat** [comment]
@YudeWang 
In order to achieving paper's performance,  is COCO2017 dataset necessary for pretrain?

**Comment by YudeWang** [comment]
@whitesockcat 
COCO pretrain is not necessary for 79% mIoU on val set.

**Comment by whitesockcat** [comment]
@whitesockcat 
Thank you very much!

**Comment by whitesockcat** [comment]
Hi, @YudeWang 
I use pretrained res101 and get **77.327%  miou** before finetune, now I'm training finetune.
I wonder is this score enough before finetune?

**Comment by whitesockcat** [comment]
Hi @YudeWang 
Sorry for bothering you again.
Is [__pycache__ folder](https://github.com/YudeWang/deeplabv3plus-pytorch/tree/master/lib/net/__pycache__) necessary for SynchronizedBN? 
And `git clone`will ignore this folder automaticly.

**Comment by YudeWang** [comment]
Just try it @whitesockcat .

**Comment by He-Chao** [comment]
@whitesockcat 
Hi, have you got 79.916% mIoU?

**Comment by whitesockcat** [comment]
@He-Chao No

**Comment by YudeWang** [comment]
[https://github.com/YudeWang/deeplabv3plus-pytorch/issues/43](url)
The similar issue is solved by re-check the dataset, please download the dataset from official webset or the link provided from README.
